tests/__sf_section_jobs.test.fish

```diff
@@ -0,0 +1,135 @@
+source $DIRNAME/spacefish_test_setup.fish
+
+function setup
+	spacefish_test_setup
+end
+
+# AppVeyor doesn't have the `killall` command, so instead of explicitly killing
+# background processes it's necessary to wait 5 seconds to terminate naturally.
+if set -q __fishtape_APPVEYOR
+  echo "~ AppVeyor detected, creating dummy killall function ~"
+  function killall
```

Comment:
Instead of creating a function, you can mock it using the mock library after sourcing it (`source $DIRNAME/mock.fish`):
```
mock killall 0 "sleep 5"
```